NETDefault/Gateway.Buckaroo/DependencyConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace Gateway.Buckaroo;
+
+public class DependencyConfiguration
+{
+    public static void Register(IServiceCollection services)
+    {
+        services.AddScoped<ISubscriptionGateway, BuckarooGateway>();
+
+        services.AddTransient<HmacDelegatingHandler>();
+
+        services.AddHttpClient<IPspClient, PspClient>()
+            .ConfigureHttpClient(c => c.DefaultRequestHeaders.Add("culture", "nl-NL"))
+            .ConfigureHttpClient(c => c.DefaultRequestHeaders.Add("channel", "Web"))

```

Comment:
Do you really need the two `ConfigureHttpClient` invocations? Or can you replace it with `.ConfigureHttpClient(c => { c.DefaultHeaders.Add([..]); c.DefaultHeaders.Add([..]) }`?